.travis.yml

```diff
@@ -0,0 +1,19 @@
+dist: trusty
+sudo: false
+
+language: node_js
+node_js:
+  - '10'
+
+before_install:
```

Comment:
remove test framework if you're not using it

Reply:
its needed for travis to build, travis failed when we took it out